### DISCUSSION TITLE
use teleport to access remote windows workstation

### DISCUSSION THREAD

**Discussion by marl-scot** [gravitational/teleport#12670] [Q&A] [answered]
Hi,
I have just started looking at teleport and would like to ask if the following is possible?

Configure and run teleport on local network with access to proxy from external.

Setup a windows 10 workstation, install teleport on it, ship machine to end user.

End user calls in with a problem.

Use teleport from inside our network to connect to the end users workstation (without using any other VPN etc)

Access remote windows workstation via powershell and/or RDP

Currently we use TeamViewer for remote support, but this is not easy to use with things like powershell etc.

Thanks

ML

**Comment by webvictim** [comment, marked as answer]
Teleport's Windows Desktop access currently requires that all machines are joined to an Active Directory domain, so if you're shipping the machines joined to AD and with an ongoing trust there then you can absolutely connect to the machine to get an RDP-like session on it in your browser. We don't support Powershell connections at the moment.

If you're not joining machines to AD, it won't work today. We're looking at relaxing the requirement for Active Directory in future.

**Reply by sbourdette** [reply]
Any news on this ?

**Reply by webvictim** [reply]
The feature request tracking this work is here: https://github.com/gravitational/teleport/issues/10166
